### PR TITLE
Add example to simple showcase test

### PR DESCRIPTION
**Pull request by kimikowang**

### CODE REVIEW

File: tests/e2e/tests/showcase/simple_showcase_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	env := test.GetEnvironment(t)
+	env.Configure(cfg)
+
+	apps := env.GetFortioApp("app=echosrv")
+
+	path := "/echo"
+	arg := "load -qps 0 -t 10s "
+	// Test Loading
+	for _, app := range apps {
+		if _, err := app.CallFotio(arg, path); err != nil {
```
Comment (ozevren) [inline tests/e2e/tests/showcase/simple_showcase_test.go]: Typo? Fotio => Fortio.

File: tests/e2e/tests/showcase/simple_showcase_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+// Reimplement TestSvc2Svc in a_simple-1_test.go
+func TestSvcLoading(t *testing.T) {
+	// This Requires statement should ensure that all elements are in runnig states
+	test.Requires(t, denpendency.FortioApps, dependency.Pilot)
+
+	env := test.GetEnvironment(t)
+	env.Configure(cfg)
+
+	apps := env.GetFortioApp("app=echosrv")
```
Comment (ozevren) [inline tests/e2e/tests/showcase/simple_showcase_test.go]: Should this simply be env.GetFortioApp("echosrv")?

File: tests/e2e/tests/showcase/simple_showcase_test.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+	path := "/echo"
+	arg := "load -qps 0 -t 10s "
```
Comment (ozevren) [inline tests/e2e/tests/showcase/simple_showcase_test.go]: Not for this checkin perhaps, but maybe we can make some of these settings strongly typed?

Comment (kimikowang) [reply tests/e2e/tests/showcase/simple_showcase_test.go]: Referring to https://github.com/istio/fortio/, there are a considerable number of options for fortio. Only of a couple of options have been used in the test though. Not sure if making these specific settings strongly typed would be appropriate or not.

File: tests/e2e/tests/showcase/simple_showcase_test.go (hunk 2 above)
Comment (kimikowang) [reply tests/e2e/tests/showcase/simple_showcase_test.go]: Just want to enforce that the `echosrv` value should be referred to key `app`. And this is how the original test code was written.

File: tests/e2e/tests/showcase/simple_showcase_test.go (hunk 1 above)
Comment (kimikowang) [reply tests/e2e/tests/showcase/simple_showcase_test.go]: Yeah thanks for pointing out the error. Will fix soon.

File: tests/e2e/tests/showcase/simple_showcase_test.go (hunk 2 above)
Comment (kimikowang) [reply tests/e2e/tests/showcase/simple_showcase_test.go]: There are two echosrv apps in the test. Other than `app`, they have other labels to distinguish them.

**Pull request merged**